airflow/contrib/operators/ssh_execute_operator.py

```diff
@@ -55,7 +55,7 @@ def __enter__(self):
         prefix = self._prefix
 
         pmktemp = ssh_hook.Popen(["-q",
-                                  "mktemp", "--tmpdir", prefix + "_XXXXXX"],
+                                  "mktemp", "-t", prefix + "_XXXXXX"],
```

Comment:
What's the preferred style for this project?
Create a check for OS and pull the flag out into a variable dependent on OS?